src/chrome/cdtpDebuggee/features/cdtpAsyncDebuggingConfigurer.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { Protocol as CDTP } from 'devtools-protocol';
+import { inject, injectable } from 'inversify';
+import { TYPES } from '../../dependencyInjection.ts/types';
+
+export interface IAsyncDebuggingConfigurer {
+    setAsyncCallStackDepth(maxDepth: CDTP.integer): Promise<void>;
+}
+
+@injectable()
+export class CDTPAsyncDebuggingConfigurer implements IAsyncDebuggingConfigurer {
```

Comment:
Let's come up with a different name besides "Configurer", I still like things named "Service"

Reply:
I like the idea of finding a better name. I'm not sure about Service though, because in some sense all Objects are services. Is there something more specific that we can use?

Reply:
I can't think of anything but I'll keep thinking about it.